ide/models/scriptfile.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+from django.utils.timezone import now
+from django.db import models
+
+from ide.models.s3file import S3File
+
+
+class ScriptFile(S3File):
```

Comment:
The motivation is that it is the superclass of both SourceFile and TestFile in the test bench branch. I don't mind changing the name if you can think of something more suitable